docs/deploy-hyperlane.mdx

```diff
@@ -1,113 +1,334 @@
 
-By the end of this guide you will have deployed and configured your Mailbox smart contract and off-chain agents, allowing developers to send interchain messages to and from your chain.
+The guide is intended for testing - by the end you’ll be able to pass messages between your custom chain and any other in the Hyperlane network.
 
-:::note
-
-This guide has been specifically made to demonstrate how to connect a chain that _already has Hyperlane on it_ to a new chain that does _not_ have Hyperlane.
+To go to production, ensure you’ve successfully pushed a registry PR (the last step) and reach back over telegram and we can add your custom chain to the core chains we have in the CLI and relay for.
 
 :::
 
-<TerminologyPartial />
+## Pre-Requisites: Chain Metadata & Funded Wallet
 
-## Overview
+Anyone can begin this quickstart guide once they have the following artifacts & assets available:
+- A new, custom, or pre-existing network of your choice, including the following metadata:
+    - A chain name, e.g. `ethereum`
+    - A chain ID, e.g. `1`
+    - A RPC URL, e.g. `https://eth.llamarpc.com`
+- A deployer wallet/EOA private key or seed phrase
+    - This EOA should be funded on your custom chain and any chain you will be passing messages to & from
+- The Hyperlane CLI npm package installed (see below)
 
-There are five steps in this guide:
+## 1) Install the Hyperlane CLI
 
-1. [<b>Set up keys</b>](#1-set-up-keys) that you will use to deploy contracts and run validators and relayer.
-2. [<b>Deploy contracts</b>](#2-deploy-contracts) to the local chain and to every remote chain with which the local chain will be able to send and receive messages.
-3. [<b>Run validators and relayer </b>](#3-run-validators-relayer) using [Kurtosis](https://cloud.kurtosis.com). Validators provide the signatures needed for the Interchain Security Modules you deployed in step 2. The relayer will deliver messages between chains you deployed contracts to.
-4. [<b>Send a test message</b>](#4-send-test-messages) to confirm that your relayer is able to deliver messages to and from each pair of chains
-5. [<b>Deploy a warp route</b>](#5-optional-deploy-a-warp-route) to send token value, not just messages, across chains
+To install the Hyperlane CLI npm package globally, run:
 
-## Getting Started
+```bash
+npm i -g @hyperlane-xyz/cli
+```
 
-## 1. Set up keys
+Verify the expected version is installed via:
 
-<SetupKeysPartial />
+```bash
+hyperlane --version
+```
 
-## 2. Deploy contracts
+To view a list of all available command, run:
 
-<DeployContractsPartial />
+```bash
+hyperlane --help
+```
 
-## 3. Run validators and relayer
+## 2) Registry: add your custom chain
 
-:::note
+Let’s create a custom chain config, run:
 
-This section covers how to 1-click deploy agents using a third party. To run the Hyperlane agent binaries yourself, follow the [Local Agents guide](guides/deploy-hyperlane-local-agents.mdx).
+```bash
+hyperlane registry init
```

Comment:
This is unrelated to this PR but I still think `registry init` is a counter-intuitive command name for "create a new chain metadata"